source/style-guide/writing/index.txt

```diff
@@ -23,6 +23,7 @@ and consistent, and that helps users accomplish their goals.
    /style-guide/writing/use-gender-neutral-pronouns
    /style-guide/writing/use-positive-statements
    /style-guide/writing/use-correct-punctuation
+   /style-guide/style/voice-and-tone
```

Comment:
Move back to Style.

Reply:
I suggested moving it because it is not easy to find as of now. Navigation in our style guide needs more work, I think you would agree.

Reply:
It's more of a reference pages and links to the related pages in the writing section. It still belongs here.